src/app/public/modules/shared/docs-tools-supportal.service.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  HttpClient
+} from '@angular/common/http';
+
+import {
+  Observable
+} from 'rxjs';
+
+import {
+  pluck
+} from 'rxjs/operators';
```

Comment:
We can't use RxJS 6 just yet; we will need to use the "side-effect" import structure until we remove `rxjs-compat` in a breaking change.
```
import {
  Observable
} from 'rxjs/Observable';

import 'rxjs/add/operators/pluck';
```

Reply:
I'm not sure I understand.  The two SPAs where I know this to be deployed and this library all have `rxjs@6.5.3` listed

- https://github.com/blackbaud/skyux-docs-tools/blob/master/package.json#L94
- https://github.com/blackbaud/skyux-popovers/blob/master/package.json#L80
- https://github.com/blackbaud/skyux2-docs/blob/master/package.json#L69

Reply:
True, but we're also installing `rxjs-compat` which adds backward compatibility for RxJS 5. (This is mainly a problem for any SKY UX 2 SPAs that are still out there that do NOT have `rxjs-compat` installed.)

Reply:
(I realize my first comment didn't give enough information about the problem; sorry about that.)